Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -2965,15 +2969,15 @@ private async Task SavePeopleMetadataAsync(IEnumerable<PersonInfo> people, Cance
 
                 if (saveEntity)
                 {
-                    (personsToSave ??= new()).Add(personEntity);
+                    if (createEntity)
+                    {
+                        CreateOrUpdateItems([personEntity], null, CancellationToken.None);
+                    }
+
                     await RunMetadataSavers(personEntity, itemUpdateType).ConfigureAwait(false);
+                    CreateOrUpdateItems([personEntity], null, CancellationToken.None);
```

Comment:
Why was the batching removed? It was originally added because it slowed down scanning

Reply:
this is mearly a quickfix. The new architecture uses ForginKeys that are actually enforced in comparison to the old DB schema. 

Unfortunatly the old code assumed on multiple places (including this one) that it can insert references before the referenced item is even present in the DB. The code inserts People for baseItems before it inserts the BaseItem.

This is no longer supported and therefor currently requires us to create the baseitem first, and then update it.